package.json

```diff
@@ -54,6 +56,7 @@
     "request": "^2.88.2",
     "should": "^13.2.3",
     "travis-cov": "^0.2.5",
+    "typescript": "^5.0.3",
     "util": "^0.12.4"
   },
   "scripts": {
```

Comment:
one more thing, so we don't forgot how to run the script to generate the types. 
add 

"scripts": {
    "build:types": "npx tsc",
}